cluster/addons/ip-masq-agent/ip-masq-agent.yaml

```diff
@@ -21,10 +21,10 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 8Mi
+            memory: 16Mi
           requests:
```

Comment:
@dnardo @dchen1107 hope can get some comments from you for this question even this is merged, thanks.